pubspec.yaml

```diff
@@ -11,10 +11,8 @@ dependencies:
   args: ^1.5.2
   async: ^2.3.0
   build_runner: ^1.7.0
-  completion: ^0.2.1+1
   glob: ^1.1.7
   io: ^0.3.3
-  meta: ">=1.2.2 <1.7.0" # Workaround to avoid https://github.com/dart-lang/sdk/issues/46142
```

Comment:
I think we may have made this a regular dependency here to avoid needing the same workaround everywhere else since `dart_dev` was such a common dependency. Would you mind keeping this just in case others happen to be relying on it transitively?

Reply:
That makes sense, will revert.